Cargo.toml

```diff
@@ -14,17 +14,21 @@ categories = ["template-engine", "value-formatting"]
 edition = "2021"
 
 [features]
+toml = ["dep:serde", "dep:toml"]
 yaml = ["dep:serde", "dep:serde_yaml"]
+preserve-order = ["toml?/preserve_order"]
```

Comment:
I think it makes more sense to call the feature `preserve_order` to keep it in line with the feature name of `toml`. What do you think ?

Reply:
I think feature names shouldn't use underscores :(

Reply:
Ahh the age old question of consistency vs what you think is right :). There [doesn't appear to be a guideline](https://rust-lang.github.io/api-guidelines/naming.html) for naming features.

Though the [examples](https://rust-lang.github.io/api-guidelines/naming.html#c-feature) seem to use dashes, so let's stick with dashes then :+1: